src/DataViews/DataView.cpp

```diff
@@ -294,18 +294,27 @@ void DataView::OnDisassemblyRequested(const std::vector<int>& selection) {
   const ProcessData* process_data = app_->GetTargetProcess();
   const uint32_t pid =
       process_data == nullptr ? app_->GetCaptureData().process_id() : process_data->pid();
+  int j = 0;
   for (int i : selection) {
     const FunctionInfo* function = GetFunctionInfoFromRow(i);
-    if (function != nullptr) app_->Disassemble(pid, *function);
+    if (function != nullptr) {
+      app_->Disassemble(pid, *function);
+      j++;
```

Comment:
I would prefer using the format that you used in CallTreeWidget:      
`if (++j > 9) break;`

Reply:
done